### DISCUSSION TITLE
Copilot incompatible with Visual Studio

### DISCUSSION THREAD

**Discussion by ne0rrmatrix** [community/community#118686] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Github copilot for visual studio updated this morning. It no longer works with Visual Studio 2022. It says it is disabled and to upgrade it. I have a current paid subscription and I can't use it. The visual studio addon does not currently work at all for me.

**Comment by hugoforte** [comment]
Same issue here

**Reply by trontronicent** [reply]
I have the same issue.
